cla_frontend/assets-src/javascripts/app/js/controllers/alternative_help.js

```diff
@@ -85,7 +85,7 @@
           $scope.getF2fDeepLink = function () {
             if ($scope.personal_details && $scope.personal_details.postcode) {
               var postcode = $window.encodeURIComponent($scope.personal_details.postcode);
-              return 'http://find-legal-advice.justice.gov.uk/search.php?searchtype=location&searchtext='+ postcode +'&searchbtn=';
+              return return 'https://find-legal-advice.justice.gov.uk/?postcode=' + postcode;
```

Comment:
🙋‍♂️ It looks like there's a double return here.

Reply:
Oh yes, well spotted. Thank you @sldblog